ethereum/env/.env.acala.testnet

```diff
@@ -1,8 +1,10 @@
 # Acala testnet env
 # Rename to .env to use with truffle migrations
 
-# Gas price generated with ./sh/acala-gas-prices.sh
-FORGE_ARGS="--legacy --with-gas-price 213314634730"
+RPC_URL=https://eth-rpc-acala-testnet.aca-staging.network
+# Note: --with-gas-price is no longer required for testnet
```

Comment:
both testnet and mainnet don't need this args.  And the `./sh/acala-gas-prices.sh` / `./sh/karura-gas-prices.sh` shell can be removed.